.github/workflows/terraform.yml

```
@@ -103,21 +103,21 @@ jobs:
 
       - name: Terraform Init
         id: init
-        run: terraform init
+        run: terraform -chdir=eks init
 
       - name: Terraform Format
-        run: terraform fmt -check
+        run: terraform -chdir=eks fmt -check
 
       - name: Terraform Validate
         id: validate
-        run: terraform validate -no-color
+        run: terraform chdir=eks validate -no-color
```

run: terraform chdir=eks validate -no-color
```suggestion
        run: terraform -chdir=eks validate -no-color
```